backend/Ript-Fitness-Backend/src/main/resources/application.properties

```diff
@@ -6,3 +6,10 @@ spring.datasource.password=${SPRING_DATASOURCE_PASSWORD}
 spring.datasource.driver-class-name=com.mysql.cj.jdbc.Driver
 spring.jpa.show-sql=true
 security.provider.1=org.bouncycastle.jce.provider.BouncyCastleProvider
+
+# Security properties for Spring
+spring.security.user.name=admin
```

Comment:
You may need to talk to Evan as he made need to add these variables to his Azure account